src/react/workflow/replication/Replication.jsx

```diff
@@ -1,85 +1,53 @@
 // @flow
```

Comment:
Let's move `src/react/workflow/replication/Replication.jsx` to `src/react/workflow/workflowTypes/Replication.jsx`.
We will have soon `src/react/workflow/workflowTypes/Expiration.jsx` for the expiration form.